### PR TITLE
Add missing red collision at door which are closed

### PR DESCRIPTION
**Pull request by monwarez**

### CODE REVIEW

**Comment by monwarez** [comment]
For the Add red collision 001-1 (103 31) it's on the map 011-1.tmx , sorry I miss type

File: maps/009-1.tmx [hunk 1]
```diff
@@ -615,7 +615,7 @@
 2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,0,0,0,0,0,0,0,0,0,0,2,0,0,2,2,2,2,2,2,2,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,2,2,2,2,2,0,0,0,2,2,2,2,2,2,2,2,2,2,2,0,2,2,0,2,2,2,2,2,2,2,2,2,2,0,0,0,0,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,
 2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,0,0,0,0,0,0,0,0,0,0,2,0,0,2,2,2,2,2,2,2,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2,2,2,2,0,0,0,2,2,2,2,2,2,2,2,2,2,2,0,0,0,0,2,2,2,2,2,2,2,2,2,2,0,0,0,0,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,
 2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,0,0,0,0,0,0,0,0,0,0,2,0,0,2,2,2,2,2,2,2,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,2,2,2,2,2,0,0,0,2,2,2,2,2,2,2,2,2,2,2,0,0,0,0,2,2,2,2,2,2,2,2,2,2,0,0,0,0,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,
-2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,0,0,0,0,0,0,0,0,0,0,2,0,0,2,2,2,2,0,2,2,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,2,2,2,2,2,0,0,0,2,2,2,2,2,2,2,2,2,2,2,0,0,0,0,2,2,2,2,2,2,2,2,2,2,0,0,0,0,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,
```
Comment (mekolat) [inline maps/009-1.tmx]: for 009-1 that door is usable, but by GMs and devs only

**Comment by mekolat** [comment]
@monwarez I'm not sure adding a collision is a good idea, as the player already can't walk through the door

**Comment by monwarez** [comment]
the player can sit on the door, making him levitate

**Comment by mekolat** [comment]
maybe try to fix the tileset then?

**Comment by monwarez** [comment]
So if I look at all the majority of house with closed door , they have a red collision on the door(for example auldsbel house) , I don't see why it is needed to fix the tileset.

**Comment by monwarez** [comment]
Maybe adding a porch at the door will stop the sitting levitate effect when doors are closed for player but not for dev/GM. 
label

**Pull request closed without merging**